### DISCUSSION TITLE
Help with varifying academic affiliation

### DISCUSSION THREAD

**Discussion by hattima-tim** [community/community#58458] [GitHub Education]
### Hi everyone,

Hello. I am having problem with varifying my academic status. My university don't provide an edu mail to undergraduate students but github requires an edu email for varifying academic status. What can I do now? Is there a way to apply for the edu pack without the edu mail? I got the edu pack without edu mail last year but this year maybe github changed their rules.

**Comment by the-officialjosh** [comment]
The typical process involves schools providing students with a unique email address using their school domain. The domain extension can vary depending on the country, such as ".edu" in the United States, ".ng" in Nigeria, or ".ca" in Canada. This allows students to have an email address specifically associated with their educational institution. Out of curiosity, what type of email domain does your school provide?

**Reply by hattima-tim** [reply]
Thanks for the reply! My university only provides an edu mail to M.S., phd, m. phil, Research students and I am a undergraduate student. The email domain it provides is `cu.ac.bd`.

**Reply by the-officialjosh** [reply]
Have you considered reaching out to fellow students at your school who are likely using the same education pack, particularly those in the computer science department? They might have encountered similar issues and could have found solutions. In my personal experience, when I faced a similar situation, I had to approach my institution's IT department to obtain the necessary email domain extension. Since this domain extension is clearly affiliated with educational institutions, starting by asking questions within your school community can be a good approach. If you haven't received any helpful responses or solutions, you might want to consider reaching out to your institution's IT department for further assistance. Alternatively, feel free to come back here with the responses you've received, and we can work together to explore possible solutions.

**Comment by OtayNacef** [comment]
check this https://github.com/orgs/community/discussions/19025#discussioncomment-2991486

**Reply by hattima-tim** [reply]
Thanks for the reply! The thread is talking about submitting documents but I am not even able to reach that part of the application process, since github is asking for an edu mail in the first page of the application form and I don't have an edu mail.
